legend-sdlc-model/src/main/java/org/finos/legend/sdlc/domain/model/project/Project.java

```diff
@@ -26,7 +26,6 @@
 
     List<String> getTags();
 
-    @Deprecated
```

Comment:
ProjectType belongs in the ProjectConfiguration rather than in the Project, so this should remain @Deprecated.

Reply:
done